yellowbrick/classifier/top_features.py

```diff
@@ -0,0 +1,253 @@
+from ..exceptions import YellowbrickValueError
+from ..style import color_palette
+
+##########################################################################
+## Top Features Visualizer
+##########################################################################
+
+class TopFeaturesVisualizer(ClassificationScoreVisualizer):
+    """
+    This takes the result of a classifier and visualizes
+    the top N positive and negative coefficients for a target class.
+    
+    Scikit-learn estimators store this information in two attributes:
+    coef_ and feature_importances_.
+    
+    Any classifier given to this visualizer must have coef_, 
+    otherwise we'll throw a "Estimator Not Implemented" warning.
+
+    Parameters
+    ----------
+    ax : matplotlib axes, default: None
+        The axes to plot the figure on. If None is passed in, the
+        current axes will be used (or generated if required).
+
+
+    N : int
+        The number of features to visualize among the positive and
+        negative coefficients.
+
+    kwargs : dict
+        Keyword arguments that are passed to the base class and may influence
+        the visualization as defined in other Visualizers.
+
+    Examples
+    --------
+    # Text Example
+    >>> from sklearn.svm import LinearSVC
+    >>> from yellowbrick.classifier import TopFeaturesVisualizer
+    >>> visualizer = TopFeaturesVisualizer(LinearSVC())
+    >>> visualizer.fit()
+    >>> visualizer.poof()
+    
+    # Numeric Example
+    >>> # numeric example here!!!!
+
+
+    Notes
+    -----
+    Text corpora need to be formatted as a vectorizer and 
+    the top features extracted before using the visualizer.
+    
+    data = corpus.data
+    target = corpus.target
+    vectorizer = CountVectorizer(stop_words='english')
+    vectorizer.fit(data)
+    X_train = vectorizer.transform(data)
+    features = vectorizer.get_feature_names()
+
+    """
+
+    def __init__(self, model, ax=None, N=20, features=None, **kwargs):
+        # TODO: Make sure model is a classifier; otherwise throw a warning.
+        # Check to see if there are N features; otherwise replace with count of features
+
+        super(TopFeaturesVisualizer, self).__init__(model=model, ax=ax, **kwargs)
+
+        self.N = N
+        self.features = features
+
+
+    def fit(self, X_train, y, **kwargs):
+        """
+        Pull in all data transform methods and apply to data here.
+        """
+
+        self.estimator.fit(X_train, y, **kwargs)
+        self.classes_ = self.estimator.classes_
+
+        return self
+
+    def score(self, class_label=None, **kwargs):
+        """
+        Extract top N features for desired class label.
+        
+        Call draw().
+        """
+
+        # raise error is no class label provided
+        if class_label is None:
+            raise YellowbrickValueError(
+                "Please provide a class label."
+                )
+
+        self.class_label = class_label
```

Comment:
My idea was to make this able to plot multiple class labels at this point instead of having to do a new visualizer, but my kicking the tires here haven't worked out, so I will move up to `__init__` and add support for when `coef_` doesn't yield the coefficients split out by classes.